src/main/java/it/gov/pagopa/pu/bff/service/AuthorizationService.java

```diff
@@ -36,29 +39,38 @@ public AccessToken postToken(String idToken) {
     log.info("Posting token for validation");
 
     return authClientImpl.postToken(
-        CLIENT_ID,
-        GRANT_TYPE,
-        SCOPE,
-        idToken,
-        subjectIssuer,
-        SUBJECT_TOKEN_TYPE,
-        null);
+      CLIENT_ID,
+      GRANT_TYPE,
+      SCOPE,
+      idToken,
+      subjectIssuer,
+      SUBJECT_TOKEN_TYPE,
+      null);
   }
 
   public void validateAdminRole(Long organizationId, UserInfo loggedUser) {
     boolean roleAdmin = isAdminRole(organizationId, loggedUser);
-    if(!roleAdmin){
+    if (!roleAdmin) {
       log.debug("Unauthorized user. [organizationId:{}]", organizationId);
       throw new AuthorizationDeniedException("Access denied on organizationId " + organizationId + " to user " + loggedUser.getMappedExternalUserId());
     }
   }
 
   public static boolean isAdminRole(Long organizationId, UserInfo loggedUser) {
-    return loggedUser.getOrganizations().stream()
-      .filter(o -> organizationId.equals(o.getOrganizationId()))
-      .findFirst()
+    return getUserOrganizationRoles(organizationId, loggedUser)
       .filter(o -> !CollectionUtils.isEmpty(o.getRoles()) && o.getRoles()
         .contains(ROLE_ADMIN))
       .isPresent();
   }
+
+  public static boolean isUserEnabledToOrganizationId(Long organizationId, UserInfo loggedUser) {
+    return getUserOrganizationRoles(organizationId, loggedUser).isPresent();
+  }
+
+  private static Optional<UserOrganizationRoles> getUserOrganizationRoles(Long organizationId, UserInfo loggedUser) {
+    return loggedUser.getOrganizations().stream()
+      .filter(o -> organizationId.equals(o.getOrganizationId()))
```

Comment:
filter also in order to retrieve only UserOrganizationRoles having not Empty roles list